tests/testFlaggedSourceSelector.py

```diff
@@ -77,18 +79,27 @@ def testSelectSources_good(self):
             self.assertIn(x, result.source_cat['id'])
 
     def testSelectSources_selected_field(self):
+        """Test the behavior of source_selected_field in run.
+
+        This test asserts that the field will specified in source selected
+        field is properly set by the source selector. We test this both for
+        sources that fail and pass our cuts.
+        """
         for i in range(5):
             add_good_source(self.src, i)
         self.src[0].set("calib_psfUsed", False)
         result = self.sourceSelector.run(
             self.src, source_selected_field=self.selected_key)
-        self.assertFalse(self.src[0].get("is_selected"))
-        self.assertTrue(self.src[1].get("is_selected"))
-        self.assertTrue(self.src[2].get("is_selected"))
-        self.assertTrue(self.src[3].get("is_selected"))
-        self.assertTrue(self.src[4].get("is_selected"))
+        for src_idx, soruce in enumerate(self.src):
```

Comment:
soruce -> source. Actually you don't use 'soruce' and therefore don't need enumerate.